slackbridge/bots.py

```diff
@@ -149,6 +151,30 @@ def nickserv_auth(self):
                 self.target_group_nick,
                 self.nickserv_password,
             ))
+    def privmsg(self, user, channel, message):
+        """
+        Set to handle if channel is own name (private chat)
+        """
+
+        nick = utils.nick_from_irc_user(user)
+        
+        if channel == self.nickname:
```

Comment:
What other scenarios do the bots get private messaged in? Mostly just curious, since I would imagine that the only private messages they get in are where the message is named after them. Maybe group messages?

Reply:
From my understanding PRIVMSG refers to both user->#channel and user->user chats contrary to what the name suggests. I can test a little more if this is actually the case since it's been working so far.